build.gradle.kts

```diff
@@ -0,0 +1,74 @@
+import org.gradle.internal.os.OperatingSystem
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    java
+    kotlin("jvm")
+}
+
+group = "com.fasterxml.jackson.module"
+version = "2.13.1-SNAPSHOT"
```

Comment:
Could we pull this from Maven rather than having it hardcoded here?

    mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v '^\['

Doing that for every Gradle run is no bueno, as that'll make Gradle super slow, we could also just grep the `pom.xml`

Reply:
I could grep it, but what if the maintainer is working on windows?

Reply:
Very good point.